terraform/test-environments/scenarios/single_local_all_inplace_upgrade.tf

```diff
@@ -0,0 +1,65 @@
+#
+# A simple local install of A2 with all supported products using the chef-automate CLI. 
+# It currently uses a pseudo-inplace upgrade.
+#
+
+module "single_local_all_inplace_upgrade" {
+  source = "github.com/chef/es-terraform//modules/cd_instance_v2"
+
+  # DNS components ( a2-all-local-inplace-upgrade-{{channel}}.cd.chef.co )
+  subdomain        = "a2-all-local-inplace-upgrade"
+  subdomain_suffix = "-${var.dns_suffix}"
+
+  # Metadata
+  meta_title       = "Single Local (Inplace Upgrade) with all supported products"
+  meta_description = "A2 stack with all supported products (using SAML) deployed locally as Habitat packages on a single host using the chef-automate CLI."
+  meta_type        = "habitat"
+
+  # AWS Instance Configuration
+  vpc            = "${var.environment}"
+  platform       = "ubuntu-16.04"
+  key_name       = "cd-infrastructure"
+  instance_type  = "m4.large"
```

Comment:
we should use m5.large.

Reply:
sorry, i didn't see this before i merged. is it still safe to change?